files-override/js/.eslintrc.cjs

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+module.exports = {
+  overrides: [
+    {
+      files: ['*.{js,mjs,cjs}'],
+      plugins: ['ember'],
+      parser: '@babel/eslint-parser',
+      parserOptions: {
+        ecmaVersion: 'latest',
+        sourceType: 'module',
+        requireConfigFile: false,
```

Comment:
this whole file is new right? can we split this into another PR? I feel like we might need to discuss the config here before adding an extra override file, I want to keep it to strictly necessary changes 👍

Reply:
ofc, yea.